src/shared/app/HomePage.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import NotFoundPage from './NotFoundPage';
+
+import PrismicReact from '../../prismic-react';
+import TextSection from './pageComponents/TextSection';
+import FullWidthImage from './pageComponents/FullWidthImage';
+import Quote from './pageComponents/Quote';
+import ImageGallery from './pageComponents/ImageGallery';
+import ImageHighlight from './pageComponents/ImageHighlight';
+import HomeBanner from './pageComponents/HomeBanner';
+
+class HomePage extends React.Component {
+  
+  constructor(props) {
+    super(props);
+    
+    this.state = {
+      notFound: false,
+      linkResolver : null,
+    };
+  }
+  
+  componentWillMount() {
+    if (typeof document !== 'undefined') document.body.classList.add('homepage');
```

Comment:
what is this @levimykel ?

Reply:
This adds the class "homepage" to the body so that the menu will be styled correctly on the homepage.